letterboxd/src/main/java/com/ataulm/letterboxd/AuthRepository.kt

```diff
@@ -1,34 +1,35 @@
-package com.ataulm.whatsnext.api.auth
+package com.ataulm.letterboxd
 
 import androidx.annotation.WorkerThread
+import com.ataulm.letterboxd.auth.AuthError
 import com.ataulm.letterboxd.auth.LetterboxdAuthApi
-import com.ataulm.whatsnext.AuthRepository
-import com.ataulm.whatsnext.LocalTokensStorage
-import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 import javax.inject.Inject
+import kotlin.coroutines.CoroutineContext
 
-class LetterboxdAuthRepository @Inject constructor(
+@LetterboxdScope
+internal class AuthRepository @Inject constructor(
```

Comment:
Since this is internal to the module, there wasn't much need to interface this out.